src/OpenVRTrackersDriver.h

```diff
@@ -57,7 +57,7 @@ class openvr::DevicesManager
 {
 public:
     DevicesManager(
-        const TrackingUniverseOrigin origin = TrackingUniverseOrigin::Seated);
+        const TrackingUniverseOrigin origin = TrackingUniverseOrigin::Standing);
```

Comment:
I am afraid this would be a breaking change for existing applications. I think we can move this parameter to the ``initialize`` method, so it should be possible to set it from a configuration file.

Reply:
In https://github.com/ami-iit/yarp-openvr-trackers/pull/12/commits/2d92454dabf540792999212eebef4776765f8265
I am passing `TrackingUniverseOrigin` as an argument, if you run the application as done normally nothing will change, but if you run it like this 

```sh
yarp-openvr-trackers --vrOrigin standing
```

It will use `TrackingUniverseOrigin::Standing`.